src/helpers/vyos_net_name

```diff
@@ -62,21 +55,22 @@ def find_available(intfs: dict, prefix: str) -> str:
 
     return f'{prefix}{len(index_list)}'
 
+
 def mod_ifname(ifname: str) -> str:
-    """ Check interface with names eX and return ifname on the next format eth{ifindex} - 2
-    """
-    if re.match("^e[0-9]+$", ifname):
-        intf = ifname.split("e")
+    """Check interface with names eX and return ifname on the next format eth{ifindex} - 2"""
+    if re.match('^e[0-9]+$', ifname):
```

Comment:
It's not a big performance problem for short strings, but I'd still do a match first, then check if the match object was `None` and used the match groups if it wasn't, instead of calling `.split()` on it, since `re.match()` result can already give us the substring.

Reply:
This is old code, just reformatted according to new rules. I believe code optimization is a separate task.